mlflow/ml-package-versions.yml

```
@@ -66,26 +66,18 @@ tensorflow:
       pip install tf-nightly
 
   models:
-    minimum: "2.3.0"
+    minimum: "2.6.0"
     maximum: "2.9.1"
     requirements:
-      "< 2.6.0": ["protobuf<4.0.0"]
       # Requirements to run tests for keras
       "== dev": ["scikit-learn", "pyspark", "pyarrow"]
     run: |
-      pytest tests/tensorflow/test_tensorflow2_model_export.py tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py
-
-      # Run tests for keras against keras-nightly that's installed with tf-nightly:
-      # https://github.com/tensorflow/tensorflow/blob/v2.6.0/tensorflow/tools/pip_package/setup.py#L110-L122
-      if [ "$PACKAGE_VERSION" == "dev" ]; then
-        pytest tests/keras/test_keras_model_export.py
-      fi
+      pytest tests/tensorflow/test_tensorflow2_model_export.py tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py test_keras_model_export.py test_keras_pyfunc_model_works_with_all_input_types.py
```

```suggestion
      pytest tests/tensorflow/test_tensorflow2_model_export.py tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py /path/to/test_keras_model_export.py /path/to/test_keras_pyfunc_model_works_with_all_input_types.py
```

Can we use absolute paths?

---

```suggestion
      pytest \
        tests/tensorflow/test_tensorflow2_model_export.py \
        tests/tensorflow/test_tensorflow2_metric_value_conversion_utils.py \
        test_keras_model_export.py \
        test_keras_pyfunc_model_works_with_all_input_types.py
```

Can we split this line for readability?